### PR TITLE
docs: spelling mistakes in the migration guide

### PR DESCRIPTION
**Pull request by puzzle-it-nu**
### 🔗 Linked issue
#2907 
### ❓ Type of change
Fix spelling mistakes

- [x] 📖 Documentation (updates to the documentation or readme)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description
Fixes what I think are some spelling mistakes.

### 📝 Checklist

- [x] I have linked an issue or discussion.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by farnabaz — APPROVED** [review]

**Comment by farnabaz** [comment]
Thanks for the PR, 
I'll wait for you to move it out of WIP

**Comment by puzzle-it-nu** [comment]
Question:
There are numbered folders in the `docs/content/docs` folder, from 1 to 7 but the number '3' occurs twice.
Is that correct?

**Comment by puzzle-it-nu** [comment]
@farnabaz I've made a mistake, commit 794379b should be reverted.
Do I make a revert commit or can I rebase and drop that commit? What is the preferred way todo this.

**Comment by puzzle-it-nu** [comment]
Things I came across but don't know what to do with:
- frontmatter vs front-matter vs Front Matter
- pascal casing in 'title' frontmatter isn't consistent
- index.md 'write content in md, yaml, json' vs 'md yaml, json, csv' in the example

Spelling in the browser :smile:  says frontmatter should either be `front matter` or `front-matter`

**Comment by puzzle-it-nu** [comment]
> Nice work @puzzle-it-nu 👍
> 
Thanks.

> > There are numbered folders in the docs/content/docs folder, from 1 to 7 but the number '3' occurs twice.
> 
> It is incorrect, and need to be fixed

Should the second '3' become '4' and so on?

> 
> > Do I make a revert commit or can I rebase and drop that commit? What is the preferred way todo this.
> 
> There isn't any preferred way, do as you prefer :)
> 

Ok, I'm in doubt whether it is really a problem. It looks more a visual indentation than that is has a function.
If the latter is the case then it can stay.

> > frontmatter vs front-matter vs Front Matter
> 
> I think we should use frontmatter all the time, as a single word since it refers to the metadata block

Ok, I'll change that where needed.

> 
> > index.md 'write content in md, yaml, json' vs 'md yaml, json, csv' in the example
> 
> Shall we add csv to index.md?

I'm in favour of that, will add it.

**Comment by puzzle-it-nu** [comment]
@farnabaz two more questions/'problems'
- point 5 'Deploy' has 3 items but they are not numbered. All the other chapters do have numbered items. My suggestion would be to number them.
- there are a dozen or so files with a frontmatter meta that contains 2 times a `title:`, looking at the content3 site it looks like the second one wins. Problem is I don't know whether the first or second one is the correct one. For example: 5.Deploy/server.md

**Comment by puzzle-it-nu** [comment]
@farnabaz about the titles makes sense, thanks

File: docs/content/docs/2.collections/1.collections.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Define Content Collections
 navigation:
-  title: Define
+title: Define
```
Comment (farnabaz) [inline docs/content/docs/2.collections/1.collections.md:4]:
```suggestion
  title: Define
```

File: docs/content/docs/2.collections/2.types.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Collection Types
 navigation:
-  title: Types
+title: Types
```
Comment (farnabaz) [inline docs/content/docs/2.collections/2.types.md:4]:
```suggestion
  title: Types
```

File: docs/content/docs/2.collections/3.sources.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Collection Sources
 navigation:
-  title: Sources
+title: Sources
```
Comment (farnabaz) [inline docs/content/docs/2.collections/3.sources.md:4]:
```suggestion
  title: Sources
```

navigation title should be indented

File: docs/content/docs/3.files/2.yaml.md [hunk 4]
```diff
@@ -49,6 +49,7 @@ Now we can query authors:
 ```ts
 // Find a single author
 const theAuthor = await queryCollection('authors')
+// TODO: should `stem` be `name`?
```
Comment (farnabaz) [inline docs/content/docs/3.files/2.yaml.md:52]: `stem` is the file name without extension. we can keep stem

File: docs/content/docs/3.files/3.json.md [hunk 5]
```diff
@@ -57,6 +57,7 @@ Now we can query authors:
 ```ts
 // Find a single author
 const theAuthor = await queryCollection('authors')
+// TODO: should `stem` be `name`?
```
Comment (farnabaz) [inline docs/content/docs/3.files/3.json.md:60]: `stem` is the file name without extension. we can keep stem

**Review by farnabaz — CHANGES_REQUESTED** [review]

File: docs/content/docs/2.collections/1.collections.md (hunk 1 above)
Comment (puzzle-it-nu) [reply docs/content/docs/2.collections/1.collections.md]: Suggestion added

File: docs/content/docs/2.collections/2.types.md (hunk 2 above)
Comment (puzzle-it-nu) [reply docs/content/docs/2.collections/2.types.md]: Suggestion added

File: docs/content/docs/2.collections/3.sources.md (hunk 3 above)
Comment (puzzle-it-nu) [reply docs/content/docs/2.collections/3.sources.md]: Suggestion added

**Comment by puzzle-it-nu** [comment]
@farnabaz added your changes and also added numbering and fixed the double '3'

There are still a few spots where the wording feels odd to me but I'm not a native english speaker so that could just me.

**Review by farnabaz — APPROVED** [review]
LGTM 👍 
Thanks

**Pull request merged**